alpha-blog/app/views/users/index.html.erb

```diff
@@ -18,8 +18,13 @@
             </h5>
             <p class="card-text"><%= pluralize(user.articles.count, "article") %></p>
             <%= link_to 'View profile', user, class: "btn btn-outline-success" %>
-             <% if logged_in? && user == current_user %>
-              <%= link_to 'Edit profile', edit_user_path(user), class: "btn btn-outline-primary" %>
+            <% if logged_in? %>
+              <% if user == current_user %>
+                <%= link_to 'Edit Profile', edit_user_path(user), class: "btn btn-outline-primary" %>
+              <% end %>
+              <% if current_user.admin? %>
```

Comment:
This logic don't match the controller. Controller say if its MY account or if I AM an admin, I can destroy the account. But here says if I am admin ONLY, i can destroy the account. Be consistent?